src/routes/tasks.ts

```diff
@@ -0,0 +1,38 @@
+import { Router } from "express"; 
```

Comment:
Why do we need this file with these routes, since the routes for the `tasks` are already added at `src/routes/project.ts`?